docs/cicd/README.md

```diff
@@ -0,0 +1,54 @@
+# CI/CD Support for Teams Application Developers
+
+TeamsFx helps automate your development workflow when building a Teams application. These documents provide some templates for you to quickly get started with CI/CD.
+
+|Tools and Templates|Description|
+|---|---|
+|[teamsfx-cli-action](https://github.com/OfficeDev/teamsfx-cli-action)|A ready-to-use GitHub Action.|
+|[github-ci-template.yml](https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd/github-ci-template.yml) and [github-cd-template.yml](https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd/github-cd-template.yml)| GitHub CI templates for a Teams app. |
+|[script-ci-template.sh](https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd/others-script-ci-template.sh) and [script-cd-template.sh](https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd/others-script-cd-template.sh)| Script templates for automation everywhere else outside of GitHub. |
+
+## CI/CD Workflow Teamplates in GitHub
+
+To add this to your repository, you will need your versions of [github-ci-template.yml](https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd/github-ci-template.yml) and  [github-cd-template.yml](https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd/github-cd-template.yml) to be located in your repository under the folder `.github/workflows`. 
+
+### Customize CI Workflow
+There are some potential changes you can make to adapt your project:
+
+1. Change how the CI flow is triggered. We default it to when pull request is created targeting dev branch.
+1. Ensure you have an npm build script, or customize the way you build in the automation code.
+1. Ensure you have an npm test script which returns zero for success, and/or change the test commands.
+
+### Customize CD Workflow
+You may want to change:
+1. How the CD flow is triggered. By default it happens when new commits are made to the `main` branch.
+1. Create GitHub repository secrets by environment to hold Azure/M365 login credentials. The table below lists all the secrets you need to create on GitHub, and for detailed usage, please refer to the GitHub Actions [README.md](https://github.com/OfficeDev/teamsfx-cli-action/blob/main/README.md).
```

Comment:
we should link directly to the GH secrets documentation here as well

Reply:
I think I added a link here, this comment might be out of date now

Reply:
I've checked that the link is added and let me resolve this comment.